pixl_imaging/src/pixl_imaging/_orthanc.py

```diff
@@ -175,6 +186,22 @@ async def job_state(self, job_id: str) -> Any:
         # See: https://book.orthanc-server.com/users/advanced-rest.html#jobs-monitoring
         return await self._get(f"/jobs/{job_id}")
 
+    async def wait_for_study_to_stabilise_or_raise(self, study_id: str) -> None:
+        """Wait for a study to become stable, or raise exception if exceeds timeout."""
+        study = await self.query_local_study(study_id)
+        is_stable = study["IsStable"]
+        start_time = time()
+        while not is_stable:
+            await sleep(10)
+            study = await self.query_local_study(study_id)
+            is_stable = study["IsStable"]
+            if not is_stable and ((time() - start_time) > self.dicom_timeout):
```

Comment:
does it make sense to use `self.dicom_timeout` here? So we wait `self.dicom_timeout` seconds for the study to be retrieved in `wait_for_job_success_or_raise `, plus an additional `self.dicom_timeout` seconds for the study to become stable

Reply:
ahhh good point, so we don't need to wait for it to stabilise? I just noticed that it wasn't stable and assumed it should be.  I'll remove these checks

Reply:
I've removed waiting for stable here as it's not needed, but we still wait for stable in the orthanc anon plugin otherwise we sometimes run into errors with exporting to xnat: 
![xnat-upload-fail](https://github.com/user-attachments/assets/845e6909-d819-444c-89ff-c2f52a9a8643)